cypress/support/commands.ts

```diff
@@ -0,0 +1,37 @@
+/// <reference types="cypress" />
```

Comment:
What does this file do?

Reply:
Yeah good pickup. It was a file automatically created upon Cypress component testing configuration. I thought I should leave it there as part of Cypress compatibility.